website/content/v1.2/introduction/what-is-new.md

```diff
@@ -4,4 +4,298 @@ weight: 50
 description: "List of new and shiny features in Talos Linux."
 ---
 
-TBD
+See also [upgrade notes]({{< relref "../talos-guides/upgrading-talos/">}}) for important changes.
+
+## Component Updates
+
+* Linux: 5.15.63
```

Comment:
```suggestion
* Linux: 5.15.64
```

would be 64 after https://github.com/siderolabs/pkgs/pull/577

Reply:
thanks, I will update it!